recipes/libattr/all/test_package/CMakeLists.txt

```
@@ -0,0 +1,9 @@
+cmake_minimum_required(VERSION 3.1)
+project(test_package)
+
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+conan_basic_setup()
+
+add_executable(${PROJECT_NAME} example.cpp)
+target_link_libraries(${PROJECT_NAME} attr)
```

```suggestion
target_link_libraries(${PROJECT_NAME} ${CONAN_LIBS})
```

CONAN_LIBS is present in conanbuildinfo.cmake. You have to use it, to validate if cpp_info was correctly configured in terms of library name.